blocks/redirects-formatter/redirects-formatter.js

```diff
@@ -0,0 +1,174 @@
+import { getLibs } from '../../scripts/utils.js';
+
+export const SELECT_ALL_REGIONS = 'Select All Regions';
+export const DESELECT_ALL_REGIONS = 'De-select All Regions';
+export const NO_LOCALE_ERROR = 'No locales selected from list';
+const INPUT_LABEL_TEXT = 'Paste source and destination URLs here:';
+const OUTPUT_LABEL_TEXT = 'Localized results appear here:';
+const PROCESS_TEXT = 'Process redirects';
+const COPY_TO_CLIPBOARD = 'Copy to clipboard';
+const INSTRUCTIONS_TEXT = 'Select the locales you require by checking the checkboxes. Paste URLs copied from an excel sheet'
+  + ' into the first input. Press "Process Redirects" to generate localized URLs to paste into redirects.xlsx. To copy your URLS,'
+  + ' press "Copy to clipboard" or select them with the cursor manually.';
+
+async function createLocaleCheckboxes(prefixGroup) {
+  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+  return Object.keys(prefixGroup).map((key) => {
+    const { prefix } = prefixGroup[key];
+    const currLocale = prefix === '' ? 'en' : prefix;
+    if (currLocale === 'langstore') return;
+    const checkbox = createTag('input', { class: 'locale-checkbox', type: 'checkbox', id: `${currLocale}`, name: `${currLocale}` });
+    const label = createTag('label', { class: 'locale-label', for: `${currLocale}` });
+    label.innerText = `${currLocale}`;
+
+    // eslint-disable-next-line consistent-return
+    return createTag('div', { class: 'checkbox-wrapper' }, [checkbox, label]);
```

Comment:
Can't you get rid of the linter issue by explicitly returning `undefined` on line 20?